semcore/color-picker/src/components/Item.tsx

```diff
@@ -52,9 +66,14 @@ export function Item(props: ItemAsProps) {
       {displayLabel && <SLabel data-value={value || '#6C6E79'}>A</SLabel>}
       <Children />
       {editable && (
-        <SCloseIcon onClick={onRemove}>
-          <CloseM color='gray-500' w={10} h={10} />
-        </SCloseIcon>
+        <>
+          <SCloseIcon tabIndex={-1} aria-hidden={true} onClick={onRemove}>
+            <CloseM color='gray-500' w={10} h={10} />
```

Comment:
If it changed, maybe we should change color to design token here?

Reply:
Yes! To `icon-primary-neutral`, as in Figma

Reply:
fixed